python/ray/cloudpickle/cloudpickle.py

```diff
@@ -405,7 +405,8 @@ def cell_set(cell, value):
 
     In Python3.7, cell_contents is writeable, so setting the contents of a cell
     can be done simply using
-    >>> cell.cell_contents = value
+
```

Comment:
This is copied directly from [cloudpickle](https://github.com/cloudpipe/cloudpickle/blob/a76f0812ccdbbd1397f36d536dc4d57b6d0557d6/cloudpickle/cloudpickle.py#L392-L447), and they don't actually call this API. Do we need to test these?

Reply:
Ah, didn't realized this was vendor code. We can just exclude these files